src/utils.jl

```diff
@@ -189,7 +189,8 @@ function getsource(source, use_mmap)
         finalize(m)
         return m2
     else
-        return slurp(source isa IO ? source : open(String(source)))
+        assert(isa(source, IO))
```

Comment:
Well, in current master, `FilePath`s are handled in this branch, so this will definitely break `FilePath` support. Better just finish up #511